static/src/javascripts/projects/common/modules/article/twitter.js

```diff
@@ -45,9 +42,17 @@ define([
             if (((scrollTop + (viewportHeight * 2.5)) > elOffset.top) && (scrollTop < (elOffset.top + elOffset.height))) {
                 fastdom.write(function () {
                     $(element).removeClass('js-tweet').addClass('twitter-tweet');
+                    // We only want to render tweets once the class has been added
+                    renderTweets();
                 });
             }
         });
+    }
+
+    function renderTweets() {
+        var scriptElement,
+            nativeTweetElements = qwery('blockquote.twitter-tweet'),
```

Comment:
Bit late to the party, but as far as I understand, running a query selector won't trigger a layout read. If you were [reading from layout](https://gist.github.com/paulirish/5d52fb081b3570c81e3a), you would need to wrap this in a `fastdom.read` to prevent forced synchronous layout. But you're not AFAICS, so no need to worry :smile: